database/settings/table.go

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package settings
+
+import (
+	"context"
+
+	"github.com/go-vela/types/constants"
+)
+
+const (
+	// CreatePostgresTable represents a query to create the Postgres settings table.
+	CreatePostgresTable = `
+CREATE TABLE
+IF NOT EXISTS
+settings (
+	id						SERIAL PRIMARY KEY,
+	compiler				JSON DEFAULT NULL,
+	queue					JSON DEFAULT NULL,
+	repo_allowlist			VARCHAR(1000),
+	schedule_allowlist		VARCHAR(1000),
+	created_at				INTEGER,
+	updated_at				INTEGER,
+	updated_by				VARCHAR(250)
```

Comment:
minor: maybe it's github ui diff, but it looks like there's some indentation issues with the types

Reply:
lol again?! ill fix it in vim, thanks